pkg/reconciler/eventlistener/resources/meta_test.go

```diff
@@ -160,10 +162,43 @@ func TestObjectMeta(t *testing.T) {
 	}}
 	for i := range tests {
 		t.Run(tests[i].name, func(t *testing.T) {
-			actualObjectMeta := ObjectMeta(tests[i].el, DefaultStaticResourceLabels)
+			actualObjectMeta := ObjectMeta(tests[i].el, FilterLabels(context.TODO(), tests[i].el.Labels), DefaultStaticResourceLabels)
 			if diff := cmp.Diff(tests[i].expectedObjectMeta, actualObjectMeta); diff != "" {
 				t.Errorf("generateObjectMeta() did not return expected. -want, +got: %s", diff)
 			}
 		})
 	}
 }
+
+func TestFilterLabels(t *testing.T) {
+
```

Comment:
It would be a bit more readable (as well as easier to add more test cases later) if this was refactored to be a table driven test. Something like:

```
for _, tc := range []struct{
  name string
  ctx context.Context
  labels map[string]string
  want map[string]string
} {{
 .... add test cases 
}}{
  if diff := cmp.Diff(tc.want, FilterLabels(tc.ctx, tc.labels)); diff != "" {
   t.Fatalf("")...
  }
}
```
```